api/RTCPeerConnection.json

```
@@ -1633,6 +1633,58 @@
             "deprecated": false
           }
         },
+        "With_callbacks": {
```

Sorry, one last nit from me

```suggestion
        "with_callbacks": {
```

---

Fixed.